composer.json

```diff
@@ -32,9 +32,6 @@
 		}
 	},
 	"autoload": {
-		"files" : [
-			"SemanticFormsSelect.php"
-		],
```

Comment:
No it will this is just removed since when it looks at the function that loads extension.json it will say it dosent exist and I think extension.json will then fail to load. It was removed from vector for that reason and other extensions too.